Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -82,11 +81,12 @@ private void calibrate() {
         isCalibrated = true;
     }
 
-    public void setCalibrate(boolean forceCalibrate) {
-        if (forceCalibrate) {
+    public void setCalibrate(boolean forceCalibrated) {
```

Comment:
Can you add some javadoc explaining what this does (what true/false means)? Just type `/**`<kbd>Enter</kbd> on the line before the method to generate the tempolate.

Reply:
In my understanding , this method is mostly used for testing, enabling us to say the elevator is calibrated without and elevator. I not sure what other application this method has. I'm not sufficiently confident in my understanding of this method, so I have not complacently filled out the javadoc.